src/demo/pages/PickersPage/PickersPage.tsx

```diff
@@ -8,15 +8,13 @@ import {
   ComponentPage
 } from '../../components/index';
 
-import { PeoplePickerTypesExample } from './examples/PeoplePicker.Types.Example';
```

Comment:
Should I remove this in it's entirety for now until we have some more specific other examples?